pdc_client/__init__.py

```diff
@@ -89,15 +95,104 @@ def read_config_file(server_alias):
     return result
 
 
+class MultiPageBeanBag(BeanBag):
+    """
+    Add this class to support to return all pages of data by default in a get request.
+
+    This class is based on the patch:
+    https://github.com/lkocman/beanbag/commit/407fff011c9e1f3883734f44a3d48fe041b2323
+    9#diff-5ad74912650ff9cf4a6754f8176f68faR336
+
+    We can use the class directly if the patch adds in the master of beanbag in the futrue.
```

Comment:
I would just remove this line. There's a typo: `futrue` → `future`.

Reply:
Changed, please review it.